### DISCUSSION TITLE
AIoT

### DISCUSSION THREAD

**Discussion by 107602505** [1101-GS4719-C/Python-Community-GS4719-C#251] [科技新知]
相信大家都知道AI人工智慧，也可能多少聽過物聯網，但有聽過AIoT嗎?
 它是物聯網與人工智慧的結合，物聯網為可以連線至網際網路的物體或「東西」，從工廠設備、汽車到行動裝置，以及智慧手錶等，還結合了感測器、軟體和其他技術的互連設備，能夠傳輸和接收其他設備資料，而人工智慧透過物聯網的數據進行機器學習，增加彼此之間的價值，相信這兩項技術的結合應用到各個產業會有很大的幫助，未來也會默默的改變我們的生活。
這篇報導用智慧冰箱舉例 https://www.inside.com.tw/feature/aiot/24214-aiot
一台真正「聰明」的冰箱，除了要連上網路，還要裝上非常多的感應器，例如重量感應器來判斷一瓶牛奶喝了多少、視覺感應器判斷什麼時候過期、味道感應器判斷食物雖沒過期但是已經腐敗了，未來若有智慧冰箱能夠提醒我們這些就能夠避免浪費食物了。

**Comment by 109802503gloria** [comment]
從智慧冰箱例子中所提到的各種感測器，看到還有很多各領域的技術需要並進，像是味道感應器應需要結合化學的氣味分子辨識的技術，感覺資料超難蒐集
